doc/zypp-NameReqPrv.1.txt

```diff
@@ -24,11 +24,11 @@ Reported matches show the packages 'internal id', 'name', 'version' and 'archite
 
 --------------------
 2431  zypper-1.9.16-22.2.x86_64  (99)@System  openSUSE  1400499579
-                                              nam: zypper
+                                              name: zypper
```

Comment:
Needs to merged manually I guess. 
The command output example in the man page here is correct (tags are '3xc:'). 
Also have .po file changes no effect if the original code string is not changed.
More than 10 years old changelog entries, ok.
Renamed variables need to be checked.
The rest seem to be typos in comments and log messages.